northstar/src/runtime/minijail.rs

```diff
@@ -509,84 +521,93 @@ impl Debug {
     }
 }
 
-/// Pipe with async tokio impls
-#[derive(Debug)]
-struct AsyncPipe {
-    non_blocking: AsyncFd<std::fs::File>,
-    blocking: RawFd,
+/// Creates a pipe with the read endpoint implementing tokio::io::AsyncRead
+async fn create_pipe() -> Result<(PipeRead, PipeWrite), Error> {
```

Comment:
Instead of a `create_pipe` there should be a `struct Pipe` that has a `fn split(self) -> (PipeLeft, PipeRight)`. 

Maybe we should move the pipe wrapper to a dedicated mod mit tests.

Reply:
I also think it would be a good idea to make a mod out of this functionality. Even better would be to get rid of it entirely :sweat_smile: